### PR TITLE
Desktop-Issue #6143 resolved. Plugins list added to about box.

### PR DESCRIPTION
**Pull request by NeerajGartia21**
This PR resolved issue #6143.
This lists all the installed plugins in the about-box.
If there are more than 20 plugins then replace the rest of them with '...'.
But when you copy it you can get all the list of plugins.

Screenshot from 2022-04-04 20-06-21

### CODE REVIEW

File: packages/lib/versionInfo.test.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+"use strict";
```
Comment (roman-r-m) [inline packages/lib/versionInfo.test.js:1]: this file shouldn't be here, it's generated by Typescript compiler

Comment (NeerajGartia21) [reply packages/lib/versionInfo.test.js:1]: I will delete that file.

Comment (NeerajGartia21) [reply packages/lib/versionInfo.test.js:1]: I have deleted the .test.js file from commit.

Comment (laurent22) [reply packages/lib/versionInfo.test.js:1]: you've deleted the TypeScript file instead...

Comment (NeerajGartia21) [reply packages/lib/versionInfo.test.js:1]: Sorry I deleted that by mistake. I'm reverting the changes and will delete the js file.

**Pull request closed without merging**